iosHyperskillApp/iosHyperskillApp/Sources/Modules/StageImplement/Modals/ProjectCompleted/StageImplementProjectCompletedModalView.swift

```diff
@@ -0,0 +1,62 @@
+import SwiftUI
+
+extension StageImplementProjectCompletedModalView {
+    struct Appearance {
+        let spacing: CGFloat = LayoutInsets.defaultInset * 2
+        let interitemSpacing = LayoutInsets.defaultInset
+    }
+}
+
+struct StageImplementProjectCompletedModalView: View {
+    private(set) var appearance = Appearance()
+
+    let stageAward: Int
+    let projectAward: Int
+
+    var onCallToActionTap: () -> Void = {}
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: appearance.spacing) {
+            Image(Images.StageImplement.ProjectCompletedModal.icon)
+
+            VStack(alignment: .leading, spacing: appearance.interitemSpacing) {
```

Comment:
@ivan-magda I think it would be great to pay our attention to that point when we will discover design before grooming to give design team feedback about such improvements

Reply:
I told them about this many times before